package main

import (
     "basic_learning/exceptions"
)

func main() {



     exceptions.GetRecover()






}
